src/mot/tracker/camera_flow.py

```diff
@@ -0,0 +1,66 @@
+import cv2
+import numpy as np
+from numpy.linalg import inv
+
+class CameraFlow():
+    '''Compute camera flow through optical flow between consecutive images
+    currently not used in the simplest version of the tracking
+    '''
+    def __init__(self):
+        self.margin_w = 200
+        self.margin_h = 20
+        self.img_shape = (768,1024)
+        self.features_mask = np.zeros(self.img_shape, dtype="uint8")
+        self.features_mask[self.margin_h:-self.margin_h, self.margin_w:-self.margin_w] = 1
+
+
+    def compute_transform_matrix(self, im_prev, im_next):
+        '''
+        Computes the transformation between two images
+
+        Arguments:
+
+        -im_prev: np array of shape self.img_shape and no color channels
```

Comment:
Portray won't be able to read "-im_prev" as a list, adding a space will allow it.

`- im_prev`